theano/tensor/nnet/corr.py

```diff
@@ -820,42 +835,42 @@ def make_node(self, kern, topgrad, shape=None):
 
     def infer_shape(self, node, input_shape):
         if self.border_mode == "half":
```

Comment:
Do you have any idea why this isn't using `get_conv_gradinputs_shape`? `get_conv_gradinputs_shape` doesn't work with `subsample`, but then why can we do it here? (Sort of a rhetoric question for another time, I suppose.)